hasjob/templates/index.html.jinja2

```diff
@@ -1,7 +1,10 @@
 {%- from "domain.html.jinja2" import org_profile, user_profile with context %}
 {% from "baseframe/forms.html.jinja2" import renderfield, widgetscripts %}
 {%- macro pagetitle() %}{% if title %}{% if domain %}Jobs at {% endif%}{{ title }} | {% endif %}{% if g.board %}{{ g.board.title }}{% if g.board.not_root %} ({{ config['SITE_TITLE'] }}){% endif %}{% else %}{{ config['SITE_TITLE'] }}{% endif %}{%- endmacro %}
-{%- if not request.is_xhr -%}
+
+{%- if embed -%}
+  {% extends "embedlayout.html.jinja2" %}
+{%- elif not request.is_xhr -%}
```

Comment:
Wait, doesn't this `if` condition do the same thing we're trying to achieve with embed? It returns an embeddable version of the page without header or footer, right?

Reply:
Yes and that's what I had done earlier. There are other parts of the page that we don't need in the embed like `flash`, campaign, `footerscripts`, there was more if/endif tags and increasing the complexity of the template.

Reply:
Also `if not request.is_xhr` doesn't include baseframe styling

Reply:
As a clean-up project, can we design templates around these use cases?

1. Full page
2. iframe (no header and footer, but includes stylesheet and scripts)
3. Fragment (just a HTML snippet)

This needs a separate ticket, and should be supported from Baseframe, so the ticket goes there.